pkg/api/meta/restmapper.go

```diff
@@ -138,7 +138,7 @@ var unpluralizedSuffixes = []string{
 }
 
 // KindToResource converts Kind to a resource name.
-func KindToResource(kind unversioned.GroupVersionKind) ( /*plural*/ unversioned.GroupVersionResource /*singular*/, unversioned.GroupVersionResource) {
+func KindToResource(kind unversioned.GroupVersionKind) (unversioned.GroupVersionResource /*plural*/, unversioned.GroupVersionResource /*singular*/) {
```

Comment:
I kind of disagree here. /*singular*/ is before a comma in the previous version. Is that a comment for the second return value? If so, why is it before a comma? If not,  what is the /*plural*/ comment for. 

I partly agree with having it the Go's declaration-syntax way though. Having trained the eyes/brain to read the inline comments after the code, it is hard to process it that way. But exactly the same could be said of Go's declaration syntax.

Having said all that, if I move the comma around to write this line as:

```
( /*plural*/ unversioned.GroupVersionResource, /*singular*/ unversioned.GroupVersionResource)
```

gofmt moves the comma around and rewrites it as:

```
( /*plural*/ unversioned.GroupVersionResource /*singular*/, unversioned.GroupVersionResource)
```

So gofmt is telling us to not write it that way? Or you think this is a gofmt bug?